rclc_examples/src/example_pingpong.cpp

```diff
@@ -144,9 +181,10 @@ int main(int argc, const char * argv[])
   // create a publisher to publish topic 'topic_0' with type std_msg::msg::String
   // my_pub is global, so that the timer callback can access this publisher.
   const char * ping_topic_name = "ping";
-  const rosidl_message_type_support_t * ping_type_support =
-    ROSIDL_GET_MSG_TYPE_SUPPORT(std_msgs, msg, String);
-
+  //const rosidl_message_type_support_t * ping_type_support =
+  //  ROSIDL_GET_MSG_TYPE_SUPPORT(std_msgs, msg, String);
+  const rosidl_message_type_support_t  ping_type_support = 
+    rclcpp::get_message_type_support_handle< std_msgs::msg::String >();

```

Comment:
Will we need rclcpp to build rclc examples?

Reply:
No.

This was a first idea for the typesupport  conversion. First, I wanted to understand, which functions are necessary to write a subscriber with callback function in C++ so that it is compatible with rclc library.The idea is to extract the necessary functions to rclc - so that we don't have the dependency of rclcpp. 

I converted this pull request to Draft.

Reply:
Cool, thanks Jan